pkg/volume/empty_dir/empty_dir.go

```diff
@@ -253,8 +258,14 @@ func (ed *emptyDir) setupTmpfs(dir string) error {
 		return nil
 	}
 
-	glog.V(3).Infof("pod %v: mounting tmpfs for volume %v", ed.pod.UID, ed.volName)
-	return ed.mounter.Mount("tmpfs", dir, "tmpfs", nil /* options */)
+	var options []string
+	// Linux system default is 50% of capacity.
+	if ed.sizeLimit != nil {
+		options = []string{fmt.Sprintf("size=%d", ed.sizeLimit.Value())}
+	}
+	glog.V(3).Infof("sizeLimit : %#v", ed.sizeLimit)
+	glog.V(3).Infof("pod %v: mounting tmpfs for volume %v with options $v", ed.pod.UID, ed.volName, options)
```

Comment:
`options` has the sizeLimit from previous line, so we don't need line 266. right?

Reply:
i already delete it in after commit

Reply:
oops. thanks!. Can you please squash the two commits into one?

Reply:
options $v?  should be %v

Reply:
orz... i will fix this